histomicstk/segmentation/label/delete_overlap.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+from .condense import condense
+from .delete import delete
+
+
+def delete_overlap(im_label, overlap_info):
+    """
+    Deletes overlapping regions from an image label based on overlap information and tile size.
+
+    Args:
+        im_label (ndarray): Image label represented as a NumPy array.
+        overlap_info (dict): Dictionary containing overlap information.
+                            It should have the following keys:
+                            'left', 'right', 'top', and 'bottom',
+                            each specifying the overlap amount in pixels.
+
+    Returns:
+        ndarray: Image label with overlapping regions deleted.
+    Note:
+        This function assumes the necessary imports, such as `np`, are already present.
+
+    """
+
+    # Compute the half of the overlap values
+    left_overlap = max(overlap_info['left'] // 2, 0)
+    right_overlap = max(overlap_info['right'] // 2, 0)
+    top_overlap = max(overlap_info['top'] // 2, 0)
+    bottom_overlap = max(overlap_info['bottom'] // 2, 0)
+
+    im_label_del = np.zeros_like(im_label)
+
+    if np.any(im_label):
+
+        # Create a border mask based on the overlap values
+        im_border_mask = np.zeros_like(im_label)
+        im_border_mask[:left_overlap, :] = left_overlap > 0
+        im_border_mask[-right_overlap:, :] = right_overlap > 0
```

Comment:
I don't think this will do what you want when `right_overlap` is 0.  In this case, you're doing `im_border_mask[0:, :] = False`, which is going to undo the `left_overlap` mask.  Further, when you do the same to the `bottom_overlap` where it is 0, it will undo all of the overlap masking.

I think you need to do:
```
if right_overlap:
  im_border_mask[-right_overlap:, :] = True
```
for right and bottom.

Reply:
I've made the changes you mentioned above. Also the order of the elements is invested since it is `[row, column]` in the NumPy array.